cmd/capibmadm/cmd/powervs/image/type.go

```diff
@@ -0,0 +1,127 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package image
+
+import (
+	"github.com/go-openapi/strfmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ISpec defines a Image.
+type ISpec struct {
```

Comment:
@KeerthanaAP, how about a more meaningful name such as `ImageSpec` and `ImageList`?

Reply:
Done. As lint is throwing the below error  when using `ImageSpec` and `ImageList` names, changed names to `ImgSpec` and `ImgList`.


Error when using `ImageSpec` and `ImageList` names:

`exported: type name will be used as image.ImageList by other packages, and that stutters; consider calling this List (revive)`